lib/modules/manager/devbox/artifacts.ts

```diff
@@ -26,6 +26,11 @@ export async function updateArtifacts({
   const execOptions: ExecOptions = {
     cwdFile: packageFileName,
     toolConstraints: [
+      // we are required to install nix because devbox spawns nix commands internally
```

Comment:
```suggestion
      // we are required to install nix because devbox spawns nix commands internally
      // https://github.com/renovatebot/renovate/discussions/35382
      // https://github.com/jetify-com/devbox/issues/2585
```

Reply:
I'm not sure the discussions there are relevant, devbox has always had this requirement it just hasn't worked in renovate until we got the previous devbox release out.